### PR TITLE
Update to run the openshift/library checks using go

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

**Comment by coreydaley** [comment]
/assign @gabemontero @adambkaplan 
I'm not 100% sure this is correct, could you all take a look at it please?

File: ci-operator/config/openshift/library/openshift-library-master.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-    tag: "3.6"
+    name: "4.6"
+    namespace: ocp
+    tag: base
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.11
+    tag: golang-1.13
```
Comment (gabemontero) [inline ci-operator/config/openshift/library/openshift-library-master.yaml:10]: this looks consistent @coreydaley with what we do in our operators .... I think it is OK ... will look at the rehearsals in a sec

**Comment by gabemontero** [comment]
ah right ... your openshift/library PR needs to merge first ... still using the python based stuff in the dockerfile 

then we can see the rehearsals

but yeah, there may be a back and forth between the 2 repos ... like I said in chat, that is OK

File: ci-operator/jobs/openshift/library/openshift-library-master-periodics.yaml [hunk 2]
```diff
@@ -46,7 +46,7 @@ periodics:
           secretKeyRef:
             key: oauth
             name: github-credentials-openshift-merge-robot
-      image: quay.io/bparees/importer:latest
+      image: registry.svc.ci.openshift.org/openshift/release:golang-1.13
```
Comment (adambkaplan) [inline ci-operator/jobs/openshift/library/openshift-library-master-periodics.yaml:49]: Referencing `registry.svc.ci.openshift.org` images directly is a no-no.

@stevekuznetsov What should we reference here?

Comment (adambkaplan) [reply ci-operator/jobs/openshift/library/openshift-library-master-periodics.yaml:49]: perhaps `openshift/release:golang-1.13`? Will the imagestream resolve correctly?

Comment (alvaroaleman) [reply ci-operator/jobs/openshift/library/openshift-library-master-periodics.yaml:49]: This should just always use the local clusters registry, is there same vanity url for that (that works in both 3.11 and 4X)?

**Comment by adambkaplan** [comment]
/lgtm

I think we can drop the WIP and merge this.

**Pull request merged**